src/tcompiler.cpp

```diff
@@ -400,6 +424,7 @@ int terra_compilerinit(struct terra_State * T) {
         return LUA_ERRRUN;
 #endif
     }
+    
```

Comment:
Let's clean up the extra newline so we don't have unnecessary diffs floating around.

Reply:
Maybe we can just start using `clang-format`?